.sonarcloud.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
this file shouldn't be included in this pull request as there were no substantive changes

Reply:
the sonarcloud properties should not be removed all together, they just shouldn't be any different from dev